docs/layers/h3-hexagon-layer.md

```diff
@@ -0,0 +1,131 @@
+<!-- INJECT:"PolygonLayerDemo" -->
+
+<p class="badges">
+  <img src="https://img.shields.io/badge/64--bit-support-blue.svg?style=flat-square" alt="64-bit" />
+  <img src="https://img.shields.io/badge/extruded-yes-blue.svg?style=flat-square" alt="extruded" />
+</p>
+
+# H3HexagonLayer
+
+The `H3HexagonLayer` renders hexagons from the [h3](https://uber.github.io/h3/) geospatial indexing system.
+
+`H3HexagonLayer` is a [CompositeLayer](/docs/api-reference/composite-layer.md).
+
+```js
+import DeckGL, {H3HexagonLayer} from 'deck.gl';
+
+const App = ({data, viewport}) => {
+
+  /**
+   * Data format:
+   * [
+   *   {
+   *     hexagonId: '882830829bfffff',
+   *     eventCount: 14030
+   *   },
+   *   ...
+   * ]
+   */
+  const layer = new H3HexagonLayer({
+    id: 'h3-hexagon-layer',
+    data,
+    extruded: true,
+    pickable: true,
+    getHexagon: d => d.hexagonId,
+    getElevation: d => d.eventCount / 3,
+    getColor: d => [255, (1 - d.eventCount / 20000) * 255, 0],
+    onHover: ({object, x, y}) => {
+      const tooltip = `${object.hexagonId}\nEvents: ${object.eventCount}`;
+      /* Update tooltip
+         http://deck.gl/#/documentation/developer-guide/adding-interactivity?section=example-display-a-tooltip-for-hovered-object
+      */
+    }
+  });
+
+  return (<DeckGL {...viewport} layers={[layer]} />);
+};
+```
+
+## Properties
+
+Inherits from all [Base Layer](/docs/api-reference/layer.md) and [CompositeLayer](/docs/api-reference/composite-layer.md) properties.
+
+### Render Options
+
+##### `coverage` (Number, optional)
+
+* Default: `1`
+
+Radius multiplier, between 0 - 1. The radius of the disk is calculated by
```

Comment:
`the disk` seems a little unclear to me, I assume this is scaling the radius of the hexagon? Could the text be updated to clarify what `the disk` is?